Created a stable-diffusion-recipe image package

0xgordian opened the pull request:
Here is the link to the package: https://github.com/Gordiancyber/recipe-stable-diffusion-2-1/pkgs/container/recipe-stable-diffusion-2-1

0xgordian committed:
Create recipe-build.yml

0xgordian committed:
Merge branch 'SaladTechnologies:main' into main

dsarfati commented on .github/workflows/recipe-build.yml:
```diff
@@ -0,0 +1,55 @@
+        id: docker_meta
+        uses: crazy-max/ghaction-docker-meta@v1
+        with:
+          images: ghcr.io/Gordiancyber/recipe-stable-diffusion-2-1
```
Why is this other repo included in this salad repo script?

0xgordian replied:
Okay. This repo relies on the availability of third-party GitHub Actions (crazy-max/ghaction-docker-meta, docker/setup-qemu-action, docker/setup-buildx-action, and docker/login-action). I’ll effect any  changes and make another PR for it if you need that. But still this is still  runnable.

0xgordian replied:
Also, this repo for stable diffusion would exist in the recipe repo as a part of it?  I'm asking so I know the placement of this repo as others would come in too.

0xgordian committed:
Update recipe-build.yml

This workflow still does the same thing as the previous workflow that was written. It builds a Docker image from a Dockerfile and pushes it to GHCR. Also, it does not use any third-party actions. it is less likely to break due to changes in the actions. It also installs the `docker-buildx` CLI plugin, which allows anyone to build multiple images.

0xgordian committed:
Update recipe-build.yml

I would recommend we use this third party as the run gets interrupted for each build that is made.
